config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/gateway/BundleElementNames.java

```diff
@@ -19,11 +19,17 @@
     public static final String ATTRIBUTE_TYPE = "type";
     public static final String ATTRIBUTE_FOLDER_ID = "folderId";
 
+    // Property names
+    public static final String PROPERTY_USER = "user";
+    public static final String PROPERTY_PASSWORD = "password";
+    public static final String PROPERTY_MIN_POOL_SIZE = "minimumPoolSize";
+    public static final String PROPERTY_MAX_POOL_SIZE = "maximumPoolSize";
+
     // Common Elements
     public static final String NAME = "l7:Name";
     public static final String PROPERTIES = "l7:Properties";
     public static final String PROPERTY = "l7:Property";
-    public static final String INT_VALUE = "l7:IntValue";
+    public static final String INT_VALUE = "l7:IntegerValue";
```

Comment:
Just want to confirm, is there l7:IntegerValue and l7:IntValue? if so should we keep both?

Reply:
I checked the xsds for the bundles and the correct is IntegerValue.
see `gateway-management-7.1.xsd` on line 907